src/main/kotlin/org/jitsi/videobridge/cc/config/BitrateControllerConfig.kt

```diff
@@ -114,6 +114,24 @@ class BitrateControllerConfig {
 
             @JvmStatic
             fun trustBwe() = trustBweProp.value
+
+            /**
+             * The property for the max resolution to allocate for the onstage
+             * participant.
+             */
+            class OnstageMaxHeightPixelsProperty : LegacyFallbackConfigProperty<Int>(
```

Comment:
Did this exist previously?  If not, just do a `SimpleProperty` which reads only from the new config

Reply:
Done that and I've also renamed the config property to `ideal` height to match the video constraint field that it pertains to. Unfortunately the new name creates a small incoherence with the max thumbnail height config property name.

To be consistent and correct, we should probably rename the thumbnail property name, but I'm worried this may break existing systems..